internal/factsengine/gatherers/sbd.go

```diff
@@ -0,0 +1,94 @@
+package gatherers
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-envparse"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/cluster"
+	"github.com/trento-project/agent/internal/factsengine/entities"
+)
+
+const (
+	SBDConfigGathererName = "sbd_config"
+)
+
+// nolint:gochecknoglobals
+var (
+	SbdConfFileError = entities.FactGatheringError{
```

Comment:
Just for consistency, I'd either always use `SBDConfig` (as you do in the gatherer name) or `SbdConf` as prefix, but not both.

My personal preference is for the above

Reply:
And I would prefix the error types with `sbd-config` to match